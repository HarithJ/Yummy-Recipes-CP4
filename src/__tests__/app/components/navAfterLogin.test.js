import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson, { shallowToJson } from 'enzyme-to-json';

import Nav from '../../../components/navAfterLogin.js'

describe("nav after login component", () => {
  const nav = shallow(<Nav />)

  it("renders properly", () => {
    expect(shallowToJson(nav)).toMatchSnapshot();
  })

  it("has a function to set search value", () => {
    expect(nav.instance().handleLogout(global.event));
  })
})
