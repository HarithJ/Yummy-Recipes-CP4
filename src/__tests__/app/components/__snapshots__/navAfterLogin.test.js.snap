// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav after login component renders properly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark justify-content-between"
>
  <span
    className="navbar-brand mb-0 h2 header-text"
    style={
      Object {
        "color": "#eee8aa",
      }
    }
  >
    Yummy
    <br />
    Recipes
  </span>
  <button
    aria-controls="yummyrecipesNavbar"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#yummyrecipesNavbar"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="yummyrecipesNavbar"
  >
    <input
      aria-label="Search"
      className="col-auto form-control mr-sm-2"
      placeholder="Search"
      type="search"
    />
    <span
      className="navbar-text ml-auto text-light"
    >
      Welcome, 
    </span>
    <button
      className="btn btn-outline-warning ml-3"
      onClick={[Function]}
      type="button"
    >
      Sign Out
    </button>
  </div>
</nav>
`;
